base/file.jl

```diff
@@ -483,7 +483,10 @@ function temp_cleanup_purge(all::Bool=true)
         if (all || asap) && ispath(path)
             need_gc && GC.gc(true)
             need_gc = false
-            rm(path, recursive=true, force=true)
+            try rm(path, recursive=true, force=true)
+            catch ex
+                @warn "temp cleanup" _group=:file exception=(ex, catch_backtrace())
+            end
         end
         !ispath(path) && delete!(TEMP_CLEANUP, path)
```

Comment:
FWIW, we've also seen CI fail here

Reply:
On this line? How would that happen? I don't think either `ispath` or `delete!` can fail.

Reply:
I know you’ve been on vacation, so it’s probably somewhere deep in your backlog to read eventually. Here’s the direct link: https://github.com/JuliaLang/julia/pull/33741#issuecomment-556145194